modular_skyrat/modules/gun_cargo/code/datums/cargo_order.dm

```diff
@@ -0,0 +1,29 @@
+/datum/supply_order/armament
+	/// The armament entry used to fill the supply order
+	var/datum/armament_entry/cargo_gun/selected_entry
+	/// The component used to create the order
+	var/datum/component/armament/cargo_gun/used_component
+	/// If it is ammo or gun
+	var/is_gun
```

Comment:
can probably make this more smart by deciding in code instead of relying on people settng it